src/compiler/generator_helpers.h

```diff
@@ -245,9 +247,9 @@ inline std::string GenerateCommentsWithPrefix(
     if (elem.empty()) {
       oss << prefix << "\n";
     } else if (elem[0] == ' ') {
-      oss << prefix << elem << "\n";
+      oss << prefix << absl::StrReplaceAll(elem, {{"$", "$$"}}) << "\n";
```

Comment:
Can you define '$' somewhere and reference it rather than using it directly? and make all generators use the same constant; https://github.com/grpc/grpc/blob/d2c2d66a03fcc2a8244361ea3479c60ac81055da/src/compiler/csharp_generator.cc#L752